src/Umbraco.Web.UI.Client/src/views/common/infiniteeditors/itempicker/itempicker.controller.js

```diff
@@ -12,24 +12,57 @@ function ItemPickerController($scope, localizationService) {
     var vm = this;
 
     vm.selectItem = selectItem;
+
+    vm.multiPicker = $scope.model.multiPicker || false;
+    vm.listType = $scope.model.listType || 'grid';
+
     vm.submit = submit;
     vm.close = close;
+    vm.isSelected = isSelected;
 
     function onInit() {
         if (!$scope.model.title) {
             localizationService.localize("defaultdialogs_selectItem").then(function(value){
                 $scope.model.title = value;
             });
         }
+
+        if (vm.multiPicker) {
+            vm.selectedItems = [];
+
+            //if (Utilities.isArray($scope.model.selectedItems)) {
+            //    vm.selectedItems = $scope.model.selectedItems;
+            //}
+            //TODO: Push selected items from model
```

Comment:
Can you explain what the idea is here? And is this a new feature, should we worry about this later maybe?

Reply:
If i recall this right I was thinking that it could be good to be able to initialize the picker with selected items, but it’s not needed for this so I drop that - forgot to remove the commented out code.